test/middlewares/generateDocs.test.ts

```diff
@@ -0,0 +1,68 @@
+import 'jest-extended'
+import { tmpdir } from 'os'
+import fs from 'fs-extra'
+import generateDocs from '../../src/middlewares/generateDocs'
+import AsyncAPIDocument from '@asyncapi/parser/lib/models/asyncapi'
+
+const TEST_ASYNCAPI_DOCUMENT = new AsyncAPIDocument({
+  asyncapi: '2.2.0',
+  info: {
+    title: 'Account Service',
+    version: '1.0.0',
+    description: 'lorem ipsum'
+  },
+  servers: {
+    test: {
+      url: 'mqtt://fake-url',
+      protocol: 'mqtt',
+    },
+  },
+  channels: {
+    'test/channel': {
+      publish: {
+        message: {
+          payload: {
+            type: 'string',
+          },
+        },
+      },
+    },
+  },
+})
+
+const CONFIG_TEST_DATA = {
+  websocket: {
+    httpServer: 'customServer',
+    adapter: 'native', // Default. Can also be 'socket.io' or a reference to a custom adapter.
+  },
+  cluster: {
+    adapter: 'redis',
+    name: 'gleeCluster',
+    url: 'redis://localhost:6379',
+  },
+  generator: {
+    folder: 'output',
+    template: 'markdown-template',
+  },
+}
+
+describe('generateDocs', () => {
+  it('should generate documentation', async () => {
```

Comment:
What is this test testing exactly? 🤔

Reply:
LOL man, I know it's testing the `generateDocs` function but my question was more about the what or how is it exactly testing it. I have the feeling that it's not testing much.

Reply:
Yeah, I have the feeling that this test is not "testing" much. I mean, it may generate something but that doesn't mean that what's generated is correct right. I know it's a unit test and not an integration test but this test feels a bit useless to me 😅

Reply:
So, are you suggesting I get rid of the test?

Reply:
Yeah but I'm a bit concerned that I'm probably not seeing the big picture here and this test should be kept, that's why I was asking. Don't have a strong opinion if it should stay or not so up to you 😄

Reply:
Yeah, don't go that far. That would be an integration test and would be a pain to maintain for no big benefit. Leave it as is, @AceTheCreator, that's fine 👍

Reply:
If you don't want to depend on the generator, you could mock `generator.generateFromString` to just return whatever you desire, and then check it created everything as expected.

So it's more tuned towards unit testing 🙂

Reply:
@jonaslagoni I could just leave it as it is now :)